lib/ansible/modules/cloud/openstack/os_nova_flavor.py

```diff
@@ -88,6 +88,11 @@
           assigned if a value is not specified.
      required: false
      default: "auto"
+   extra_specs:
+     description:
+        - Metadata dictionary
+     required: false
+     default: None
```

Comment:
Along with the doc change, please force the version requirement in the code itself too.
We use StrictVersion for that, have a look at https://github.com/ansible/ansible/blob/829c0b8f6284ec37e36f1454d7d3640415239cb2/lib/ansible/modules/cloud/openstack/os_network.py#L186 .